packages/settingeditor/style/base.css

```diff
@@ -148,96 +202,283 @@
   left: 6px;
 }
 
-#setting-editor .jp-SettingsRawEditor .jp-Inspector {
+.jp-SettingsRawEditor .jp-Inspector {
   border-top: 2px solid var(--jp-layout-color2);
   min-height: var(--jp-private-settingeditor-debug-height);
   max-height: var(--jp-private-settingeditor-debug-height);
 }
 
-#setting-editor
-  .jp-SettingsRawEditor
-  .jp-Inspector.jp-SettingsDebug
-  .jp-RenderedHTMLCommon {
+.jp-SettingsRawEditor .jp-Inspector.jp-SettingsDebug .jp-RenderedHTMLCommon {
   padding: 2px 5px 2px 0;
   width: 100%;
 }
 
-#setting-editor
-  .jp-SettingsRawEditor
-  .jp-Inspector.jp-SettingsDebug
-  .jp-RenderedHTMLCommon
-  p {
+.jp-SettingsRawEditor .jp-Inspector.jp-SettingsDebug .jp-RenderedHTMLCommon p {
   text-align: right;
 }
 
-#setting-editor .jp-SettingsTableEditor {
-  border: 1px solid var(--jp-brand-color1);
-  margin: 0;
-  padding: 0;
+.jp-SettingsPanel fieldset input,
+.jp-SettingsPanel fieldset select,
+.jp-SettingsPanel fieldset textarea {
+  font-size: var(--jp-content-font-size2);
+  border-color: var(--jp-input-border-color);
+  border-style: solid;
+  border-radius: 5px;
+  border-width: 1px;
+  padding: 6px 8px;
+  background: none;
+  color: var(--jp-content-font-color0);
+}
+
+.jp-SettingsPanel .checkbox label {
+  cursor: pointer;
+}
+
+.jp-SettingsPanel button[type='submit'] {
+  display: none;
+}
+
+.jp-SettingsPanel .form-group {
+  padding: 0 26px 16px 10px;
+  display: flex;
+}
+
+.jp-SettingsPanel #root__description {
+  display: none;
+}
+
+.jp-SettingsPanel .checkbox {
+  padding-top: 20px;
+}
+
+.jp-SettingsPanel fieldset {
+  border: none;
 }
 
-#setting-editor .jp-SettingsTableEditor legend {
+.jp-SettingsPanel .jp-SaveSettingsBanner {
+  position: absolute;
+  bottom: 0;
+  padding: 20px;
+  width: 100%;
+  background: var(--jp-border-color3);
+}
+
+.jp-SettingsPanel .jp-SaveSettingsBanner button {
+  box-shadow: none;
+  outline: none;
+  border: none;
   color: var(--jp-brand-color1);
-  font-size: 70%;
-  font-weight: bold;
-  margin-left: 15px;
-  height: var(--jp-private-settingeditor-legend-height);
+  font-size: var(--jp-ui-font-size1);
+  cursor: pointer;
+}
+
+.jp-SettingsPanel .jp-SaveSettingsBanner button:hover {
+  color: var(--jp-brand-color0);
+}
+
+.jp-SettingsPanel .form-group.small-field:hover {
+  background: var(--jp-border-color3);
+}
+
+.jp-SettingsPanel i.glyphicon {
+  display: none;
+}
+.jp-SettingsPanel .btn-add::after {
+  content: 'Add';
+}
+.jp-SettingsPanel .array-item-move-up::after {
+  content: 'Move Up';
+}
+.jp-SettingsPanel .array-item-move-down::after {
+  content: 'Move Down';
+}
+.jp-SettingsPanel .array-item-remove::after {
+  content: 'Remove';
+}
```

Comment:
This is not blocking (if you do not plan to address it before we merge, let's create a new issue to track it), but:

> _Using an `::after` pseudo-element to add content is discouraged, as it is not reliably accessible to screen readers._

https://developer.mozilla.org/en-US/docs/Web/CSS/::after#accessibility_concerns

CC @isabela-pf

Reply:
Also, we won't be able to translate these strings if they are defined in CSS.

Reply:
Oh great point - there is a way around this, I can try to get that into this PR but if not I can definitely make an issue

Reply:
@krassowski Just pushed a change that should address this issue. Let me know if there are any issues